src/vegas_mode/render_pipeline.py

```
@@ -228,19 +233,16 @@ def render_frame(self) -> bool:
                         from PIL import Image as _Image
                         blank = _Image.new('RGB', (self.display_width, self.display_height))
                         self.display_manager.image = blank
                         self.display_manager.update_display()
-                    except (ImportError, OSError, RuntimeError, ValueError, TypeError, MemoryError) as exc:
-                        logger.error(
-                            "Failed to push blank frame at cycle end "
-                            "(display=%dx%d): %s",
-                            self.display_width, self.display_height, exc
-                        )
+                    except Exception:
+                        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)